app/assets/stylesheets/internal/layout.scss

```diff
@@ -1,112 +1,112 @@
-@import '../crayons';
-@import '../scaffolds';
-@import '../variables';
-
-#siteConfigBodyContainer {
```

Comment:
@ludwiczakpawel removing these styles breaks the ui. 
For example on the site config page `.site-config__required` is used for the required tag, etc. Same with the others.
It doesn't look like they've been replaced in some other way?

Reply:
Let me double check that.

Reply:
You were right, good catch. I thought it would cause a harm. Anyway, I brought some of the styling back to the [admin.scss](https://github.com/forem/forem/pull/10015/files#diff-111bd41e9b25b9eaafc015184274028d) file and will gradually remove/replace that with Crayons stuff.

Reply:
@ludwiczakpawel great but there's still more that we're using, these I know are being used offhand without ay further research:
```
#siteConfigBodyContainer {
  padding: 20px;
}

#siteConfig .card-header {
  cursor: pointer;
}

#siteConfig .card-header__toggle {
  font-size: 12px;
  color: var(--secondary);
}

#siteConfig .notice {
  margin: 20px 0;
  font-weight: bold;
  color: $dark-purple;
  font-size: 14px;
}

#profileFields .card-header {
  cursor: pointer;
}

.profile__group-toggle {
  text-decoration: underline;
  cursor: pointer;
  font-size: 12px;
  color: grey;
  margin: 1em 0;
  text-align: end;
}
```
Perhaps we should add them all back to avoid regressing the UI and then in a subsequent PR we can remove those that we're sure are not being used and move others to crayons if thats possible.

Reply:
```
#siteConfigBodyContainer {
  ...
}
```

This has been replaced with some new styling already. It should still look good.

```
#siteConfig .card-header {
  ...
}

#profileFields .card-header {
  ...
}
```

We have button that toggles the card body, so even if the whole header is clickable it doesn't need a pointer cursor. I quite intentionally removed it, because eventually ONLY button should toggle the content, especially in some other cases we may have more buttons than just a "toggle" (remember our designs for Profile Fields? I know it's a different view, but the same UI pattern eventually)

```
#siteConfig .notice {
  ...
}
```

This has been replaced with `.crayons-notice` element.

.profile__group-toggle {
  ...
}

You're right - I'll bring that back in.

Reply:
@ludwiczakpawel sounds good. Thanks. Regarding the toggles, it makes complete sense especially with the new profile design. One thing to note though is we don't use a button on the inner card headers, hence perhaps we should add a cursor pointer to at least the text?

<img width="943" alt="Screenshot 2020-08-26 at 16 59 45" src="https://user-images.githubusercontent.com/2786819/91320233-96058200-e7bd-11ea-922a-bd0e9b580f00.png">
 
Happy with this being in a subsequent PR though. 

Also made a note elsewhere in the code about .notice and a git merge :) https://github.com/forem/forem/pull/10015/files#r477367526

Reply:
Yeap, agree - I'll make it a separate PR later on.

FYI: the changes I've made with current PR are basically very first step for Admin visual improvements :) More are coming!